gluten-core/src/main/scala/io/glutenproject/backendsapi/SparkPlanExecApi.scala

```diff
@@ -444,14 +444,15 @@ trait SparkPlanExecApi {
               frame.frameType.sql
             )
             windowExpressionNodes.add(windowFunctionNode)
-          case wf @ NthValue(input, offset: Literal, _) =>
+          case wf @ NthValue(input, offset: Literal, ignoreNulls: Boolean) =>
             val frame = wExpression.windowSpec.frameSpecification.asInstanceOf[SpecifiedWindowFrame]
             val childrenNodeList = new JArrayList[ExpressionNode]()
             childrenNodeList.add(
               ExpressionConverter
                 .replaceWithExpressionTransformer(input, attributeSeq = originalInputAttributes)
                 .doTransform(args))
             childrenNodeList.add(LiteralTransformer(offset).doTransform(args))
+            childrenNodeList.add(LiteralTransformer(Literal(ignoreNulls)).doTransform(args))
```

Comment:
You could use makeLiteral to get an expression node.

Reply:
Will fix it in a separate PR. Thanks!